pages/common/pt.md

```diff
@@ -0,0 +1,23 @@
+# pt
+
+> Platinum Searcher. A code search tool similar to ag. Respects .gitignore.
```

Comment:
Please move the second sentence to a new heading. Put ag within backticks. 

We can also remove the last sentence. It does not need to be in a heading.